aiida/cmdline/verdilib.py

```diff
@@ -947,18 +947,6 @@ def _check_db_name(self, dbname, method=None, **kwargs):
         return dbname, create
 
 
-class Runserver(VerdiCommand):
```

Comment:
Yes. One question, I will remove the wsgi.py file but should we conside to provide such a file for the new restapi? This requires to include some extra requirements.